language-server/src/main/scala/scala/meta/languageserver/ScalametaLanguageServer.scala

```diff
@@ -58,6 +58,16 @@ class ScalametaLanguageServer(
 
   private val toCancel = ListBuffer.empty[Cancelable]
 
+  private def loadAllSemanticdbsInWorkspace() = {
+    Files
+      .walk(workspacePath.toNIO)
+      .filter(
+        path =>
+          Files.isRegularFile(path) && path.toString.endsWith(".semanticdb")
+      )
+      .forEach(path => semanticdbSubscriber.onNext(AbsolutePath(path)))
```

Comment:
We might want to do this on parallel, could be a lot of semanticdb files.

Reply:
right, that makes sense -> #11